selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/testrunner/JavaTestPluginTestNgTest.java

```diff
@@ -140,7 +142,13 @@ public void shouldExecuteTestMethodsSuccessfully() throws InterruptedException {
     assertTrue(pluginConsole.getAllNamesOfMethodsMarkedDefinedStatus(PASSED).size() == 1);
     editor.goToCursorPositionVisible(30, 17);
     menu.runCommand(RUN_MENU, TEST, TEST_NG_TEST_DROP_DAWN_ITEM);
-    pluginConsole.waitMethodMarkedAsFailed("shouldFailOfAppAnother");
+    try {
+
+      pluginConsole.waitMethodMarkedAsFailed("shouldFailOfAppAnother");
+    } catch (TimeoutException ex) {
+      // remove try-catch block after issue has been resolved
+      fail("Known issue https://github.com/eclipse/che/issues/7338");
```

Comment:
It's better to pass **ex** variable to `fail()` method to have additional info about the catching exception.